queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```

Comment:
- [ ]   `LIKE '%mon%'` means Finds any values that have "mon" in any position, but According to the requirements we need to Find all animals whose name ends with "mon"., So it would be better to use `LIKE '%mon'` 💯

```sql
SELECT * FROM animals WHERE name LIKE '%mon';
```

![image](https://user-images.githubusercontent.com/45231276/196305791-fdf272d4-3663-4b27-975d-38ecda93a1a5.png)